source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,154 @@
+ - Fixed an issue where the OAuth 2.0 deprecation notice was still displayed in the system console.
+ - Fixed an issue where clicking on a reply post time stamp in the global threads inbox opened two right-hand side panels.
+
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in the ``config.json``:
+    - ``PostPriority``, to add an option to select a message priority label for root posts.
+    - ``AllowSyncedDrafts``, to add an option to display a centralized page for draft messages.
+ - Under ``ExperimentalSettings`` in the ``config.json``:
+    - ``PatchPluginsReactDOM`` to enable the patching of the React DOM library when loading web app plugins so that the plugin uses the version matching the web app.
+
+### API Changes
+ - The resumable uploads API was exposed to plugins.
+ - Added a new API endpoint ``POST /api/v4/ldap/users/:user_id/group_sync_memberships`` to add (or re-add) users to all of their default teams and channels for all of the groups they're a member of.
+ - Added two new URL parameters to the ``GET /api/v4/groups`` endpoint to get the ``ChannelMemberCount`` for a group.
+ - Added new API endpoints ``POST /api/v4/users/:user_id/posts/:post_id/ack`` and ``DELETE /api/v4/users/:user_id/posts/:post_id/ack``.
+ - Added a new API endpoint ``POST /api/v4/groups/:group_id:/restore``.
+ - A new API method ``RegisterCollectionAndTopic(collectionType, topicType string) (error)`` was added to the Plugin API and the following hooks. This API method is in beta, subject to change, and not covered by our backwards compatibility guarantee.
+    - ``UserHasPermissionToCollection(c *Context, userID, collectionType, collectionId string, permission *model.Permission) (bool, error)``
+    - ``GetAllCollectionIDsForUser(c *Context, userID, collectionType string) ([]string, error)``
+    - ``GetAllUserIdsForCollection(c *Context, collectionType, collectionID string) ([]string, error)``
+    - ``GetTopicRedirect(c *Context, topicType, topicID string) (string, error)``
+    - ``GetCollectionMetadataByIds(c *Context, collectionType string, collectionIds []string) (map[string]model.CollectionMetadata, error)``
+    - ``GetTopicMetadataByIds(c *Context, topicType string, topicIds []string) (map[string]*model.TopicMetadata, error)``
+
+### Database Changes
+ - Added a new Database table ``PostAcknowledgements``.
+ 
+### Websocket Event Changes
+ - Added new websocket events ``post_acknowledgement_added`` and ``post_acknowledgement_removed``.
+
+### Go Version
+ - v7.6 is built with Go ``v1.18.1``.
+
+### Known Issues
+ - Email notifications sometimes look broken when email batching is enabled [MM-48521](https://mattermost.atlassian.net/browse/MM-48521).
```

Comment:
Given that this issue is noted as fixed above, do we want to keep it mentioned here too as a known issue? Or is this case a bit different because it's specifically batched email notifications?

Reply:
This bug hasn't been fixed yet.

Reply:
The bug is now fixed and I've removed it from the known issues.